library/src/main/java/no/finntech/capturandro/GalleryHandler.java

```diff
@@ -58,28 +61,28 @@ void handle(Uri selectedImage, String filename, CapturandroCallback.ImageHandler
                 int columnIndex = cursor.getColumnIndex(MediaStore.MediaColumns.DATA);
 
                 if (isPicasaAndroid3Image(selectedImage) || imageIsRemote(cursor)) {
-                    imageHandler.onGalleryImportStarted(filename);
-                    fetchPicasaAndroid3Image(selectedImage, filename, cursor);
+                    imageHandler.onGalleryImportStarted(importId);
```

Comment:
Outside the scope of this PR, but it would be nice if we could make these more generic.

Maybe we could have one generic "GetImage" async task that handles started/success/error. And everything could use that.